content/docs/cn/getting-started/1-quick-start.md

```diff
@@ -1,13 +1,77 @@
 ---
-title: Quick Start
-contributors: [rigor789, eddyverbruggen, damain]
+title: 快速开始
+contributors: [rigor789, eddyverbruggen, damain, ikoevska, Pythonfo]
```

Comment:
You can remove everyone else, and only keep yourself in the contributors list, since these are the contributors for the english version.

Reply:
I just translate this, but they are writer. I just think so. OK, I will remove.